diff --git a/node_modules/@typespec/ts-http-runtime/dist/commonjs/util/sha256.js b/node_modules/@typespec/ts-http-runtime/dist/commonjs/util/sha256.js
index 6966285..3bd15a6 100644
--- a/node_modules/@typespec/ts-http-runtime/dist/commonjs/util/sha256.js
+++ b/node_modules/@typespec/ts-http-runtime/dist/commonjs/util/sha256.js
@@ -4,7 +4,13 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.computeSha256Hmac = computeSha256Hmac;
 exports.computeSha256Hash = computeSha256Hash;
-const node_crypto_1 = require("node:crypto");
+let node_crypto_1;
+try {
+    node_crypto_1 = require("node:crypto");
+}
+catch (error) {
+    node_crypto_1 = require("crypto");
+}
 /**
  * Generates a SHA-256 HMAC signature.
  * @param key - The HMAC key represented as a base64 string, used to generate the cryptographic HMAC hash.
